bin/bento

```diff
@@ -69,10 +70,10 @@ class Options
         },
         argv: templates_argv_proc
       },
-      fix: {
```

Comment:
I'm wondering if we should deprecate/warn but not remove the 'fix' CLI command in favor of making it an alias of 'normalize' at least for a little while. I may be being more conservative than is needed though. I feel like as small as it is, it is technically a 'breaking' change.